backend/package.json

```diff
@@ -20,6 +20,8 @@
     "db:publish": "bash ./util/publish-db.sh",
     "docs": "bash ./util/publish-docs.sh",
     "sequelize-cli:source": "npm run build:setenv:dev && npm run build && ./node_modules/.bin/sequelize --config src/database/sequelize-cli-config.ts --migrations-source-path src/database/migrations",
```

Comment:
It's not actually migrate:dev since it's missing that `db:migrate` parameter at the end. Should I add it/add another script `migrate:dev` or...?

Reply:
Yeah -- Don't have a strong opinion about this on a second thought (Since we're syncing to models in dev for now) We can keep it w/o a migrate:dev script and use the sequelize-cli wrappers if we wanna do stuff on dev anyways

Reply:
Cool - can we resolve this conversation or I should change something anyways?